### PR TITLE
Unpinned a selected filter from agent dashboard

### PR DESCRIPTION
**Pull request by Mayons95**
Hi team ! I create this PR to add a new test case to cover the unpin functionality from the agent module dashboard.
I'm working on reusing some existing functionality. I'll add more info as soon I advance with the development

Screenshot from 2022-11-09 14-45-13

### CODE REVIEW

**Comment by Mayons95** [comment]
image

I found this selector that it's broken. I'm working to fix it and run the complete test execution

**Comment by Mayons95** [comment]
Test it's working for WZD

File: test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Feature: Unppined filter from agent - Dashboard
+
+  As a Wazuh user
+  I want to unppined a filter
+  in order to aplly it across the modules
```
Comment (asteriscos) [inline test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature:5]: There's a typo in "apply"

File: test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    And The user navigates to the agent dashboard
+
+  @filter @actions
+  Scenario Outline: The user add and pin new filer - across the modules - from dashboard to event page <Module Name>
+    When The user navigates to agentModule <Module Name>
+    And The user adds a new filter
+    And The user checks filter label is added
+    And The user pins a filter
+    And The user checks if the filter is displayed
+    And The user unppined the selected filter
```
Comment (asteriscos) [inline test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature:18]: There's a typo in "unpinned"

File: test/cypress/cypress/integration/step-definitions/filters/the-user-unppined-the-selected-filter-when.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { When } from 'cypress-cucumber-preprocessor/steps';
+import { elementIsVisible, clickElement, getSelector} from '../../utils/driver';
+import { FILTERS_PAGE as pageName} from '../../utils/pages-constants';
+const pinnedFilter = getSelector('pinnedFilter', pageName);
+const unPinFilterAction = getSelector('unPinFilterAction', pageName);
+
+
+When('The user unppined the selected filter', () => {
```
Comment (asteriscos) [inline test/cypress/cypress/integration/step-definitions/filters/the-user-unppined-the-selected-filter-when.js:8]: There's a typo in "unpinned"

**Review by asteriscos — CHANGES_REQUESTED** [review]
Please review a few typos.

**Comment by Mayons95** [comment]
WZD report generated:
[Uploading WZD-09-11.pdf…]()

File: test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature (hunk 1 above)
Comment (Mayons95) [reply test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature]: Done

File: test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature (hunk 2 above)
Comment (Mayons95) [reply test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature]: Done

File: test/cypress/cypress/integration/step-definitions/filters/the-user-unppined-the-selected-filter-when.js (hunk 3 above)
Comment (Mayons95) [reply test/cypress/cypress/integration/step-definitions/filters/the-user-unppined-the-selected-filter-when.js]: Done

File: test/cypress/cypress/integration/features/filters/remove-filter-from-agent-module-events.feature [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Feature: Validate that the added filter label is remove after click remove filter option agent modules Dashboard
```
Comment (AlexRuiz7) [inline test/cypress/cypress/integration/features/filters/remove-filter-from-agent-module-events.feature:1]: This sentence is so hard to understand

File: test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature [hunk 5]
```diff
@@ -0,0 +1,32 @@
+Feature: Unppined filter from agent - Dashboard
+
+  As a Wazuh user
+  I want to unppined a filter
```
Comment (AlexRuiz7) [inline test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature:4]: I want to **unpin**

File: test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  in order to apply it across the modules
+  Background:
+    Given The wazuh admin user is logged
+    And The user navigates to the agent page
+    And The user navigates to the agent dashboard
+
+  @filter @actions
+  Scenario Outline: The user add and pin new filer - across the modules - from dashboard to event page <Module Name>
```
Comment (AlexRuiz7) [inline test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature:12]: The user add**s** and pin**s** **the** new filter

File: test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    When The user navigates to agentModule <Module Name>
+    And The user adds a new filter
+    And The user checks filter label is added
+    And The user pins a filter
+    And The user checks if the filter is displayed
+    And The user unpinned the selected filter
```
Comment (AlexRuiz7) [inline test/cypress/cypress/integration/features/filters/unpinned-filter-from-agent-dashboard.feature:18]: And the user unpin**s**

**Review by asteriscos — APPROVED** [review]
LGTM :heavy_check_mark:

**Review by AlexRuiz7 — APPROVED** [review]

**Pull request merged**